### DISCUSSION TITLE
Padding on mobile view

### DISCUSSION THREAD

**Discussion by miked98** [squidfunk/mkdocs-material#6646] [Q&A] [answered]
Hello,

I got a lot of feedback that some padding would be nice on mobile view. 

I was able to change that with custom CSS:
```
@media(max-width:1150px) {
  .md-content__inner{
    padding: 0.6rem;
  }
}
```

IOS and Android users gave me feedback like that.

I can see that it looks fine on https://squidfunk.github.io any ideas why?

**Comment by kamilkrzyskow** [comment, marked as answer]
Hello @miked98,
please be more specific about what you mean. Using your custom CSS adds padding on top of the existing margin, I'm not sure why it looks fine on the theme's documentation and not on your own.

<details open>
<summary> Example video where I toggle the CSS: </summary>

https://github.com/squidfunk/mkdocs-material/assets/34622465/b8d06ced-fdc5-4e94-990c-c130910f5949

</details>

**Reply by miked98** [reply]
I'm sorry it seems that it is an issue with my custom css then.     
On my site I do not have any padding/margin to sides on mobile view.

Thank you very much for your fast response!
